src/test/scala/me/jeffmay/neo4j/client/ws/TestGlobal.scala

```diff
@@ -0,0 +1,25 @@
+package me.jeffmay.neo4j.client.ws
+
+import com.typesafe.config.{Config, ConfigFactory}
+import me.jeffmay.util.akka.{NonBlockingCountingThrottler, TestGlobalAkka}
+
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try}
+
+object TestGlobal {
+
+  lazy val config: Config = Try(ConfigFactory.load()) match {
+    case Success(loaded) => loaded
+    case Failure(ex) =>
+      println("\u001B[31m" +
+        s"""
+           |Could not load required configuration for tests. Encountered exception:
+           |${ex.getMessage}
+           |
+           |Run docker/start.sh to get a printout of the required configs.
+           |""".stripMargin + "\u001B[0m")
+      sys.exit(1)
+  }
+
+  lazy val requestThrottler: NonBlockingCountingThrottler = TestGlobalAkka.throttler(5.5.seconds)
```

Comment:
FAIL: I left this class out of the commit because it was kind stupid. This will cause compiler errors.